src/app/TopBar.module.css

```
@@ -0,0 +1,217 @@
+  h1 {
+    display: none;
+    margin: 0;
+    -webkit-user-select: none;
+    user-select: none;
+  }
+
+  /* Only style and show it from 601px and up */
+  @media (min-width: 37.5625rem) {
+    h1 {
+      display: block;
+      font-family: 'Futura', 'Trebuchet MS', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Futura" (font-family-name-quotes)

(font-family-name-quotes)